internal/tasks/resource_scrape.go

```diff
@@ -95,7 +96,11 @@ func (c Context) ScrapeNextResource(assetType db.AssetType, maxScrapedAt time.Ti
 	if err != nil {
 		return err
 	}
-	resourceScrapedTime := c.TimeNow()
+
+	now := c.TimeNow()
+	fmt.Println(now)
+	res.CheckedAt = now
+	res.ScrapedAt = &now
```

Comment:
You need to set `res.ScrapeErrorMessage = ""` here, otherwise errors won't clear up. That particular type of glitch is still fresh for me because I fixed the same problem in Keppel today. :)